packages/core/apps/core-apps-server-internal/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { CoreAppsService } from './src';
+export {
+  registerBundleRoutes,
```

Comment:
These are needed for the integration tests in `src/core/server/integration_tests`. 
We can decide as a team how to handle the integration tests during the migration cleanup phase. The pending tooling changes should give us a bit more freedom in package structure and inteneral dependencies.

Reply:
NIT: In similar cases for other packages, I added a comment to 'separate' the 'real' imports from the integration-test-only imports.

E.g:

https://github.com/elastic/kibana/blob/92ca42f007492dbda15ef60c73b6a7e944a9f917/packages/core/saved-objects/core-saved-objects-server-internal/index.ts#L17-L22